nss-day-cohort-41/tabloidmvc-oysters-rockefeller-rose
# List/Update via UserProfiles (x3 Tickets)

#UserProfile -Pull Request (x3 TICKETS):

->CRUD has been achieved for Categories SECTION:
..TICKET: (#24)View All Users 
..TICKET: (#25)View a single Users Profile
..TICKET: (#26)Update a UserProfiles “UserType”

No Dependencies, so far. The changes are to the Controller for Accounts…. I have added methods that allow the user Profile to list the current users by all “user types”, as well as pull the details on a single user. Finally, a profile can be adjusted/updated via a drop down for “UserType”.

Fixes #
## Type of change or Added file
-  Added Files: ACCOUNTController.cs, UserTypeRespository.cs, IUserTypeRepository.cs, UserTypeViewModel.cs
 - ACCOUNT VIEWS( DETAILS , EDIT,  INDEX).cshtml

- New feature:  This process has been tested and verified without issue/bug.  You can navigate via Menu Management/Navigation....
 
# Testing Instructions
Upon running "TabloidMVC":
-Login via admin email (admin@example.com), and then review the NAV BAR OPTIONS
-Click "Userz"
-A List and View of the current Profiles List will appear with the following properties (#24)
-Via the Details button, a list of the user specific properties will be displayed, and an edit button will display (#25)
-Via the Edit option a user can change the administrative privileges (via dropdown on Usertype) or cancel their edit and be returned to the index (#26)

# Checklist:
- [ yes ] My code follows the style guidelines of this project
- [ yes ] I have performed a self-review of my own code
- [ yes ] I have commented my code, particularly in hard-to-understand areas
- [ yes ] My changes generate no new warnings or errors
- [ yes ] I have added test instructions that prove my fix is effective or that my feature works
